store/core/pom.xml

```diff
@@ -48,8 +48,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>8</source>
```

Comment:
1.8 ?

Reply:
8 is same as 1.8, as this is passed to javac
https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javac.html